portfolio/src/main/webapp/slideshow_script.js

```diff
@@ -0,0 +1,55 @@
+//sliding
+var slideIndex = 1;
+var timeout;
```

Comment:
Can we name this a bit more specifically? What is the timeout for?

Reply:
I changes this to showTimeout